src/instruments/src/SD/Common/HydraulicsProvider.tsx

```diff
@@ -0,0 +1,40 @@
+import { useSimVar } from '@instruments/common/simVars';
+import React from 'react';
+
+interface HydraulicSystemAvailable {
+    available: boolean
+}
+
+interface HydraulicsContext {
+    G: HydraulicSystemAvailable;
+    Y: HydraulicSystemAvailable;
+    B: HydraulicSystemAvailable;
+}
+const HydraulicContext = React.createContext<HydraulicsContext>({
+    G: { available: false },
+    Y: { available: false },
+    B: { available: false },
+});
+
+export const HydraulicsProvider = ({ children }) => {
+    const [greenPress] = useSimVar('L:A32NX_HYD_GREEN_PRESSURE', 'number', 1000);
+    const [yellowPress] = useSimVar('L:A32NX_HYD_YELLOW_PRESSURE', 'number', 1000);
+    const [bluePress] = useSimVar('L:A32NX_HYD_BLUE_PRESSURE', 'number', 1000);
+
+    const hydraulicContext: HydraulicsContext = {
+        G: { available: greenPress > 1450 },
+        Y: { available: yellowPress > 1450 },
+        B: { available: bluePress > 1450 },
+    };
```

Comment:
I think it should be a hysteresis 1450 / 1750 like ecam hyd ?
This should come as a boolean from hydraulic wasm to simplify the life of everyone....

Reply:
The HYD ECAM SD page doesn't use any hysteresis for the system low pressure, it's just a simple comparison. If pressure<1450, the color is in amber, and else, it's green.

Reply:
That could very well be. I've moved the values introduced in #4870 to a shared location. I would suggest creating a separate PR which modifies the WHEEL, F/CTL and HYD page to use calculated values coming from `systems.wasm` in a separate PR.